src/overview/components/VisitAsListItem.css

```diff
@@ -33,41 +35,54 @@
 
 .thumbnail {
     display: block;
-    max-width: 240px;
-    width: auto;
-    height: auto;
+    height: 100%;
+    width: 100%;
```

Comment:
It was observed when the page width was decreased the div with the screenshot was shrinking more when no screenshot was present and less when there was a picture in the div(screenshot was present). 
I researched a bit more and found that when the width of the container is decreased in flex, it fits according to the content given in the div. Therefore the height and width are set to 100% for when there is no screenshot the div occupies the same space as that of when there is a screenshot.